graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -1,14 +1,38 @@
-# Copyright 2017-present Kensho Technologies, LLC.
+# Copyright 2018-present Kensho Technologies, LLC.
+import codecs
 import datetime
 import os
 import random
+import re
 import sys
 
-from ... import __version__
 from .animals import get_animal_generation_commands
 from .species import get_species_generation_commands
 
 
+#  https://packaging.python.org/guides/single-sourcing-package-version/
+#  #single-sourcing-the-version
+
+
+def read_file(filename):
+    """Read package file as text to get its version"""
+    # intentionally *not* adding an encoding option to open
+    # see here:
+    # https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
+    top_level_directory = os.path.dirname(os.getcwd())
```

Comment:
`cwd` isn't constant, so this code is brittle -- use `__file__` because this file's location is fixed, then make it relative to that